### PR TITLE
Revert "Removes borg combat spinning"

### PR DESCRIPTION
**Pull request by ZombiesVsPlants**
## About The Pull Request
Reverts austation/austation#2278
## Why It's Good For The Game

Now that I've tested the nerfed spins out I can confidently say they are cringe.
Now that I'm back from my holiday let me take apart this PR and tell you why it is bad

> cyborgs are meant to be good at doing one specific task designated by their module. security and emagged/syndi borgs are for combat
Cyborgs were, are, and will always be shit at combat
Non lethally in combat:
An Engi borg has
- A flash (Countered by advanced sunglasses/welding helmet/literally spray painted eyewear)
- and spin (Comparable to a human shove)

that's it
sure there are other unique things you could do like drag around beepsky, maybe drag around a thing of n2o, but that means you have time/resources to get them in the first place. in the heat of the moment, with a human greytider with advanced sunglasses (easily gotten) and a flash that, if it gets you, almost GUARANTEES you will die, spinning is your only option. (Which, by the way, put you into range of the flash and if you get a lagspike or are too slow you WILL get flashed)

Humans can:
Shove 
aggressive grab (Tablestun./throw)
any number of non-lethal items they can get their hands on:
- stun stick/stun prod/telescopic baton
- flash
- disabler
- dragnet
- and more
Can ingest chems 
- Stun resistance (rip borg who thought he had 2 more seconds)
- meth speed (borgs can't speed up apart from rare VTEC module)
laserpointers
and more

once again, cyborgs, while stronger than a naked human don't compare to a human that has items, *as it always has been.*

The only cyborgs that can compare to a kitted out human (or hell just a human with a flash) is emagged (Traitor) or security which, on austation, is locked behind research and mats and is thus very rare.

Furthermore just because a module is made to do a specific purpose doesn't mean they can't try and be robust and sitting there, knowing you can do nothing to a greytider/traitor/shitter who is harming a human just cause they have advanced sunglasses is cringe.

> To the people (hal) that will inevitably say "but I need muh spin stun or greytider kill me11!"

First:
ezgif-3-f9fd1bb35cbe
Second: 
Cyborgs have hard counters, one of which is produced on mass in a vulnerable area that stuns you enough to kill you with a fire extinguisher and can be refilled with LIGHT BULBS
They can be range countered by a laserpointer that has high tier parts (or hell level 2 is still dangerous)
So yes, I DO need spin stun in order to have just a chance against anyone with these low tier items,

> Please remember you are using external tools to gain a combat advantage.

bruh what?
Proof
i've been using stacked text boxes ever since I got bwoinked for it cause like, I don't want to be banned? I have not been using auto hotkey for ages and all the spin stuns are from me moving my mouse fast and using a semblance of skill.

TLDR: Re-adds spin stuns as they are sorely needed by borgs, reverts the ided PR.

Changelog
🆑
balance: Borg spins stun again
balance: You can buckle/be buckled to a borg instantly again
balance: Borgs can no longer buckle stunned/unconscious bodies 
/🆑

### CODE REVIEW

**Comment by thelaughingbomb** [comment]
If the paralyze time was changed from 6 seconds to 3 or even 4 seconds that would be a substantial enough nerf that doesnt make borg spins completely useless.

**Comment by coolsurf6** [comment]
Knockdown makes more sense. Paralysing for 6 second was pretty fucking powerful and fighting borg's without hur dur magic flash was a pain in the ass. 





And stop getting cause using macros lol.

**Comment by ZombiesVsPlants** [comment]
bro not even using macros

**Comment by MrLostman** [comment]
think we need to nerf borgs even more for not being grateful that only spin stun was removed

**Comment by TheFakeElon** [comment]
^^

**Comment by TheFakeElon** [comment]
Borgs. Should. Be. Worse. Than. Humans. In. Their. non-specialised. tasks.

You know what borg is good for validhunting/dealing with shitters? The _security_ borg. You showed that your preferred strategy for spin stunning was queuing up a large amount of *spin text boxes and using that to have instant stuns when needed. There’s a big problem when borg speen becomes a mix between Krav Maga and the power fist.

**Comment by MCterra10** [comment]
Imma go full oranges

**Comment by ZombiesVsPlants** [comment]
@TheFakeElon Borgs were/are/will always be worse at combat than carbons due to lack of hands, as I said in the pr.

The only reason that it seems so powerful is because I've gotten robust as borg and have learnt over 100 hours of playing borg how to spin stun effectively.
This is nerfing because someone has gotten good at their main role, I'm sure you remember when I was learning it and got completely robusted by you when validhunting constantly.

Your whole security borg argument doesn't make sense as you are completely reliant on robotics being not dead, not braindead, miners aren't braindead.

It is like removing the armory and telling sec they can get their validhunting tools after science researches advanced guns.

Please read the pr
Or
Let sec modules be roundstart again.

**Comment by ZombiesVsPlants** [comment]
Apologies for spelling errors I'm on mobile
